Eastern-Research-Group/mywaterway
# Feature/state boundaries layer

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3561185

## Main Changes:
* Added in the state boundaries layer. I used the following layer for this: https://services.arcgis.com/P3ePLMYs2RVChkJx/ArcGIS/rest/services/USA_States_Non_Generalized/FeatureServer 
* Updated the legend ordering of the supplemental layers to more closely reflect the ordering they show up in the layer list widget. I left the waterbodies, monitoring stations and permitted discharger ordering alone, since it appears that we were matching the ordering of the toggles for these.

## Steps To Test:
1. Navigate to the community page
2. Toggle the state boundaries layer
3. Expand the legend
4. Verify the state boundaries display on the map and show up in the legend
